WalletConnectSharp.Core/Network/AESCipher.cs

```diff
@@ -64,9 +64,9 @@ public async Task<string> DecryptWithKey(byte[] key, EncryptedPayload encryptedD
         if (encoding == null)
             encoding = Encoding.UTF8;
 
-        byte[] rawData = encryptedData.data.FromHex();
-        byte[] iv = encryptedData.iv.FromHex();
-        byte[] hmacReceived = encryptedData.hmac.FromHex();
+        byte[] rawData = encryptedData.data.HexToByteArray();
+        byte[] iv = encryptedData.iv.HexToByteArray();
+        byte[] hmacReceived = encryptedData.hmac.HexToByteArray();
```

Comment:
whoops. i also fixed this as part of #49 but it should be a straightforward rebase